crates/bevy_ui/src/node_bundles.rs

```diff
@@ -77,8 +79,9 @@ impl Default for NodeBundle {
 ///
 /// # Extra behaviours
 ///
-/// You may add the following components to enable additional behaviours
+/// You may add the following components to enable additional behaviours:
```

Comment:
(the same as for `SpriteBundle`): I think it should be clear whether `ImageScaleMode` and `TextureAtlas` can be used together or not. If not, here it should say "either component" or similar.

Reply:
Added a note.